Sources/Algorithms/isSorted.swift

```diff
@@ -0,0 +1,51 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the Swift Algorithms open source project
+//
+// Copyright (c) 2020 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+//
+//===----------------------------------------------------------------------===//
+
+//===----------------------------------------------------------------------===//
+// isSorted()
+//===----------------------------------------------------------------------===//
+
+
+extension Sequence where Element: Comparable {
+    public func isSorted() -> Bool {
+        /// Returns Bool, indicating whether a sequence is sorted
```

Comment:
To follow the Swift documentation style, these doc comments should start with “Returns a Boolean value indicating whether...”